modules/data.land/R/extract_NEON_veg.R

```diff
@@ -0,0 +1,56 @@
+#' extract_NEON_veg
+#' @title extract_NEON_veg
+#' @name extract_NEON_veg
+#' 
+#' @param lon site longitude, passed from ic_process
+#' @param lat site latitude, passed from ic_process
+#' @param start_date "YYYY-MM-DD", used to download NEON datasets for desired time period
+#' @param end_date "YYYY_MM_DD", used to download NEON datasets for desired time period
+#' @param ... Additional parameters
+#' 
+#'
+#' @return veg_info object to be passed to extract_veg within ic_process
+#' @author Alexis Helgeson and Michael Dietze
+#' @export
+#' 
+#' @importFrom rlang .data
+#' 
+#' @examples start_date = as.Date("2020-01-01") 
+#' end_date = as.Date("2021-09-01")
+
+extract_NEON_veg <- function(lon, lat, start_date, end_date, ...){
+
+#Set store_dir to point to dietzelab/neon_store
+store_dir = "/projectnb/dietzelab/neon_data"
```

Comment:
This was fine for development, but you can't hardcode a directory within a package that could be used by anyone on any computer.